### ISSUE TITLE
Error about missing `cargo-sweep` in post job cleanup.

### ISSUE THREAD

**Issue by ilyvion** [Leafwing-Studios/cargo-cache#42]
I got the following in the post-job cleanup of `cargo-cache`:
```
Post job cleanup.
Post job cleanup.
Restoring timestamp from state.
Using timestamp: {"secs_since_epoch":1735565006,"nanos_since_epoch":191004420}.
Sweeping unused build files.
/home/runner/work/_actions/BD103/cargo-sweep/v1/dist/post/index.js:5682
                    throw new Error(`Unable to locate executable file: ${tool}. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also check the file mode to verify the file is executable.`);
                          ^

Error: Unable to locate executable file: /home/runner/.cargo/bin/cargo-sweep. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also check the file mode to verify the file is executable.
    at Object.<anonymous> (/home/runner/work/_actions/BD103/cargo-sweep/v1/dist/post/index.js:5682:27)
    at Generator.next (<anonymous>)
    at fulfilled (/home/runner/work/_actions/BD103/cargo-sweep/v1/dist/post/index.js:5535:58)

Node.js v20.18.0
Warning: Input 'save-always' has been deprecated with message: save-always does not work as intended and will be removed in a future release.
A separate `actions/cache/restore` step should be used instead.
See https://github.com/actions/cache/tree/main/save#always-save-cache for more details.

Post job cleanup.
/usr/bin/tar --posix -cf cache.tzst --exclude cache.tzst -P -C /home/runner/work/extol_image_font/extol_image_font --files-from manifest.txt --use-compress-program zstdmt
Cache Size: ~879 MB (921695332 B)
Cache saved successfully
Cache saved with key: <...>
```

Manually installing `cargo-sweep` fixed the problem, but it feels like this action should be making sure that `cargo-sweep` is installed, not I. I used the latest released version (or, to be specific, `Leafwing-Studios/cargo-cache@v2`).

**Comment by BD103** [comment]
Hi, thanks for the bug report! The bug you encountered was fixed in `v2.5.0`, but it looks like I never updated the `v2` tag. Whoops! 🤦

`v2` has now been set to `v2.5.0`, so this shouldn't happen anymore. Could you please confirm that this error is now fixed in your CI?

**Comment by ilyvion** [comment]
I can confirm that your update to the tag has fixed my issue. Thank you. 😊
